### PR TITLE
Fixed max_send_limit_bytes to be in all kafka outputs

### PR DESCRIPTION
**Pull request by kyleboyer-optum**

### CODE REVIEW

**Comment by repeatedly** [comment]
Thanks. I will check after back from kubecon.

File: lib/fluent/plugin/out_kafka.rb [hunk 1]
```diff
@@ -228,11 +228,15 @@ def emit(tag, es, chain)
         partition = (@exclude_partition ? record.delete('partition'.freeze) : record['partition'.freeze]) || @default_partition
         message_key = (@exclude_message_key ? record.delete('message_key') : record['message_key']) || @default_message_key
 
-        value = @formatter_proc.call(tag, time, record)
-
-        log.trace { "message will send to #{topic} with partition_key: #{partition_key}, partition: #{partition}, message_key: #{message_key} and value: #{value}." }
+        record_buf = @formatter_proc.call(tag, time, record)
+        record_buf_bytes = record_buf.bytesize
+        if @max_send_limit_bytes && record_buf_bytes > @max_send_limit_bytes
+		  log.warn "record size exceeds max_send_limit_bytes. Skip event:", :time => time, :record => record
```
Comment (ganmacs) [inline lib/fluent/plugin/out_kafka.rb]: [nits] Could you use space to indent to match other codes?

Comment (kyleboyer-optum) [reply lib/fluent/plugin/out_kafka.rb]: @ganmacs Resolved!

File: lib/fluent/plugin/out_kafka.rb [hunk 2]
```diff
@@ -228,17 +228,21 @@ def emit(tag, es, chain)
...
-	rescue Kafka::BufferOverflow => e
-	  log.warn "BufferOverflow occurred: #{e}"
-	  log.info "Trying to deliver the messages to prevent the buffer from overflowing again."
-	  producer.deliver_messages
-	  log.info "Recovered from BufferOverflow successfully`"
-	end
+        record_buf = @formatter_proc.call(tag, time, record)
+        record_buf_bytes = record_buf.bytesize
+        if @max_send_limit_bytes && record_buf_bytes > @max_send_limit_bytes
+                  log.warn "record size exceeds max_send_limit_bytes. Skip event:", :time => time, :record => record
```
Comment (ganmacs) [inline lib/fluent/plugin/out_kafka.rb]: This line and L235 should be indented.

Comment (repeatedly) [reply lib/fluent/plugin/out_kafka.rb]: I will fix this later.

**Pull request merged**